package.json

```diff
@@ -49,4 +51,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
\ No newline at end of file
+}
```

Comment:
I really think you should fix your vscode so that it doesn't delete newlines and makes these files ILLEGAL.

Reply:
Is this file illegal if it has an extra linebreak?

Reply:
I think I added it? In any case, it has a newline now.